global.json

```diff
@@ -7,7 +7,7 @@
     "Microsoft.Build.NoTargets": "3.3.0"
   },
   "sdk": {
-    "version": "6.0.400",
+    "version": "6.0.300",
```

Comment:
Why are we moving to the previous version?

Reply:
Not sure, I think the reason was that GTKSharp workflow was only supported up to 6.0.300, @parhamsaremi can you elaborate?

Reply:
> GTKSharp workflow was only supported up to 6.0.300

this should be done upstream in GtkSharp.
I opened an issue: https://github.com/GtkSharp/GtkSharp/issues/400

Reply:
but what's wrong with going back to .300? I mean, if it works, why not support an older version?

Reply:
anyway, it will be removed in later versions of maui:
https://github.com/dotnet/maui/pull/10492

Reply:
workload package is here: https://github.com/orgs/GtkSharp/packages/nuget/package/GtkSharp.NET.Sdk.Gtk.Manifest-6.0.400

Reply:
@parhamsaremi can we revert the 400->300 changes now then? ^

Reply:
@knocte I'll work on it.

Reply:
@lytico @jsuarezruiz FYI, your feedback has been addressed